src/app/settings/organization-plans.component.ts

```diff
@@ -197,6 +197,16 @@ export class OrganizationPlansComponent implements OnInit {
     }
 
     async submit() {
+        if (this.onlyOrgPolicyBlock) {
+            return;
+        } else {
+            const policies = await this.policyService.getAll(PolicyType.OnlyOrg);
+            this.onlyOrgPolicyBlock = policies.filter(policy => policy.enabled).length > 0;
```

Comment:
I think you can use `policies.some(policy => policy.enabled)` here to be more efficient.

Reply:
fixed